static/app/views/replays/detail/page.tsx

```diff
@@ -40,7 +40,7 @@ export default function Page({
   isVideoReplay,
 }: Props) {
   const title = replayRecord
-    ? `${replayRecord.id} — Session Replay — ${orgSlug}`
+    ? `${replayRecord.user.display_name} — Session Replay — ${orgSlug}`
```

Comment:
just wanted to confirm that user.display_name always exists?

Reply:
good point - i added a fallback to Anonymous User